__tests__/hooks/useGroupInfo.test.js

```diff
@@ -694,10 +763,6 @@ describe('useGroupInfo should run correctly', () => {
     expect(result.current.newParticipant).toEqual([]);
   });
 
-  it('onReportGroup should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.onReportGroup();
-  });
   it('handlePressContact type group hould run correctly', async () => {
     const {result} = renderHook(() => useGroupInfo(), {wrapper});
     await result.current.handlePressContact({user_id: '123', user: {name: 'agita'}});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test for `onReportGroup` has been removed in the new hunk. If this function is no longer part of the component, then it's fine. Otherwise, ensure that there is a test covering this functionality.
```diff
-  it('onReportGroup should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.onReportGroup();
-  });
  it('handlePressContact type group hould run correctly', async () => {
    const {result} = renderHook(() => useGroupInfo(), {wrapper});
    await result.current.handlePressContact({user_id: '123', user: {name: 'agita'}});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->